secure_message_v2/models/models.py

```diff
@@ -13,7 +13,7 @@ class Base(DeclarativeBase):
 class Thread(Base):
     __tablename__ = "thread"
 
-    id: Mapped[UUID] = mapped_column(primary_key=True)
+    id: Mapped[UUID] = mapped_column(primary_key=True, default=uuid4)
```

Comment:
Although easier to generate and manage, using a randomly generated 128 bit PK over a sequential integer may have performance and indexing consequences in the future.